swift_scality_backend/diskfile.py

```diff
@@ -109,11 +118,20 @@ def put(self, metadata):
             raise
 
         self._release_conn()
-        metadata['name'] = self._name
+
+        metadata_to_put = copy.copy(metadata)
+        metadata_to_put['name'] = self._name
+        metadata_to_put['ETag'] = self._md5sum.hexdigest()
+
+        metadata_to_put.update({
+            'df': metadata,
```

Comment:
Are `df` and `mf` names we choose, or enforced by Swift?

Reply:
These are chosen by us, I'm open to alternative naming suggestions :)